packages/core/src/render3/component_ref.ts

```diff
@@ -535,7 +535,7 @@ export class ComponentRef<T> extends AbstractComponentRef<T> {
     if (ngDevMode && !hasSetInput) {
       const cmpNameForError = stringifyForError(this.componentType);
       let message = `Can't set value of the '${name}' input on the '${cmpNameForError}' component. `;
-      message += `Make sure that the '${name}' property is declared as an input using the @Input() decorator or the input() function.`;
+      message += `Make sure that the '${name}' property is declared as an input using the input() or model() function or the deprecated @Input() decorator.`;
```

Comment:
```suggestion
      message += `Make sure that the '${name}' property is declared as an input using the input(), model() function or the @Input() decorator.`;
```

We haven't deprecated the decorator.